### ISSUE TITLE
Teaching idea

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/training-material#5588]
The https://hemingwayapp.com tells you how complex your writing is.

Would be neat to link this to the Training material testing to indicate to the authors how complex their text is, and similarly link this in the metadata as another indicator of difficulty

**Comment by hexylena** [comment]
Have also considered this, adding e.g. fleisch-kinkaid but the concern here is it is bordering on scientific writing, we occasionally need to explain complex concepts and that takes a lot of words that get flagged by common algorithms as too complex. In theory it is good, in practice, you'd have to be very careful.

there is also more acceptance/tolerance of different English dialects in the GTN than other places, e.g. french English, and German English, so, we'd want to be careful if we did implement something that it doesn't unnecessarily target ESL speakers

**Comment by nomadscientist** [comment]
Good call. I wonder if it would be useful as a 'behind the scenes' thing? Like if it was not public on the tutorial itself, but instead generated as part of the linting/testing in Github process. Then they click a link and it takes them to a page (such as the Hemingway app) which uses different colours / symbols to highlight sections. It may be they can ignore it, or it might be a quick way to spot places where things get a bit tricky. So they can still make a choice, rather than be blocked by it, or it be in the masthead for learners (agree, that's a bad idea, I retract that now!)
